src/pages/instances/forms/InstanceFormMenu.tsx

```diff
@@ -87,7 +88,9 @@ const InstanceFormMenu: FC<Props> = ({
               />
               <MenuItem label={GPU_DEVICES} {...menuItemProps} />
               <MenuItem label={PROXY_DEVICES} {...menuItemProps} />
-              <MenuItem label={OTHER_DEVICES} {...menuItemProps} />
+              {useSupportedFeatures().hasMetadataConfiguration ? (
```

Comment:
generally we use custom hooks outside of the return of a react component, let call `useSupportedFeatures()` above.